db/re/item_db.txt

```diff
@@ -679,7 +679,7 @@
 1314,F_Tomahawk_C,Tomahawk,5,2,,0,200,,1,0,0x000444A2,63,2,34,4,0,0,7,{ bonus bAtkEle,Ele_Wind; },{},{}
 1315,F_Right_Epsilon_C,Light Epsilon,5,1,,0,229,,1,0,0x000444A2,63,2,34,4,1,0,7,{ bonus bAtkEle,Ele_Holy; bonus bStr,10; },{},{}
 1316,Adventure_Axe,Adventure Axe,5,0,,0,100,,1,0,0x00040420,63,2,2,1,1,0,6,{},{},{}
-1317,Academy_Axe,Academy Axe,5,0,,1600,130,,1,1,0x00040420,63,2,2,1,1,1,6,{},{},{}
+1317,Academy_Axe,Academy Axe,5,0,,1600,130,,1,1,0x00040420,63,2,2,1,1,1,6,{ bonus bMaxHP,200; bonus bAspdRate,5; bonus bAspdRate,BaseLevel/10; bonus bMaxHP,(-40*(BaseLevel/10)); },{},{}
```

Comment:
Missing the unbreakable bonus(`bonus bUnbreakableWeapon,1;`) and the required equip level(30).
It looks like the base level is only related to the aspd decrease, but that is just my interpretation of the korean translation.

Reply:
@Lemongrass3110 In DivinePride says MaxHP -40 too, It just used _comma_ to not repeat. _(In my interpretation)_

But I had misunderstood too, I had understood so:

_Increase Aspd Rate in 1% every 10 base levels but decrease MaxHP in 40._

Then how would it look?

I've did it:

```
1317,Academy_Axe,Academy Axe,5,0,,1600,130,,1,1,0x00040420,63,2,2,30,1,1,6,{ bonus bMaxHP,200; bonus bAspdRate,5; bonus bUnbreakableWeapon,1; bonus bAspdRate,(-1*(BaseLevel/10)); bonus bMaxHP,(-40*(BaseLevel/10)); },{},{}
```

I didn't know if this is correct:

```
bonus bAspdRate,(-1*(BaseLevel/10));
```